app/locales/sk.json

```diff
@@ -48,6 +48,8 @@
   "Share on": "Zdieľajte na",
   "NB!": "Upozornenie!",
   "This page is currently not affiliated with the authorities in any way.": "Táto stránka je nezávislou iniciatívou dobrovoľníkov, nesúvisí so štátnými orgánmi.",
+  "This page is not affiliated with the authorities.":"Táto stránka nie je prepojená s orgánmi.",
```

Comment:
"Táto stránka je nezávislou iniciatívou."

Reply:
updated.. what a cool looking language!